avocado/core/future/settings.py

```diff
@@ -45,6 +45,7 @@
 import os
 
 from ..settings_dispatcher import SettingsDispatcher
+from pkg_resources import resource_filename
```

Comment:
Hi @sathnaga the code looks good to me, thanks.

Only one small thing: can you please run an `isort` here? Imports are out of order.

Reply:
@sathnaga @beraldoleal shouldn't we also include the `conf.d` content? As it is, it reduces the gap when compared to the `avocado.core.settings` based behavior, but still lacks support for the individual configuration files in `conf.d`.

Reply:
@clebergnu I would love to see some explicit "include" implementation. Where on config file we could have an `include` directive.

But yes, I understand that this might be not so easy for now, so I'm ok with a hardcoded "conf.d" include.